app/services/retry_task_service.rb

```diff
@@ -0,0 +1,51 @@
+class RetryTaskService < TaskService
+  LOST_TIME_INTERVAL = ClowderConfig.instance["LOST_TIME_INTERVAL"].to_i * 60
```

Comment:
@hsong-rh I think a better variable name could be **INACTIVE_TASK_REMINDER_TIME**